### DISCUSSION TITLE
2.41.0 / 2022-12-20

### DISCUSSION THREAD

**Discussion by roidelapluie** [prometheus/prometheus#11747] [Announcements]
* [FEATURE] Relabeling: Add `keepequal` and `dropequal` relabel actions. #11564
* [FEATURE] Add support for HTTP proxy headers. #11712
* [ENHANCEMENT] Reload private certificates when changed on disk. #11685
* [ENHANCEMENT] Add `max_version` to specify maximum TLS version in `tls_config`. #11685
* [ENHANCEMENT] Add `goos` and `goarch` labels to `prometheus_build_info`. #11685
* [ENHANCEMENT] SD: Add proxy support for EC2 and LightSail SDs #11611
* [ENHANCEMENT] SD: Add new metric `prometheus_sd_file_watcher_errors_total`. #11066
* [ENHANCEMENT] Remote Read: Use a pool to speed up marshalling. #11357
* [ENHANCEMENT] TSDB: Improve handling of tombstoned chunks in iterators. #11632
* [ENHANCEMENT] TSDB: Optimize postings offset table reading. #11535
* [BUGFIX] Scrape: Validate the metric name, label names, and label values after relabeling. #11074
* [BUGFIX] Remote Write receiver and rule manager: Fix error handling. #11727

<hr /><em>This discussion was created from the release <a href='https://github.com/prometheus/prometheus/releases/tag/v2.41.0'>2.41.0 / 2022-12-20</a>.</em>

**Comment by himir11285** [comment]
is there any date for End Of Life OR End Of Support for Prometheus 2.41.0 released ?
